### PR TITLE
E1812. Write unit tests for on_the_fly_calc.rb

### PR DESCRIPTION
**Pull request by ashwinrisbood**
First commit

### CODE REVIEW

**Comment by ysgeer1** [comment]
Thank you for the feedback.
Climate issues are fixed. Still 2 remaining, not sure how to fix.
Issue 1 - fixed: round_sym, grades_by_rounds and total_num_of_assessments are removed from lines where the code can assign them the values.
Issue 2: we never get that issue when we run tests, but the added line should fix the problem.

**Comment by Winbobob** [comment]
Thank you for your quick response! The remaining 2 code climate issues are fine. I will look into your tests.

File: app/models/on_the_fly_calc.rb [hunk 1]
```diff
@@ -60,11 +61,11 @@ def scores(questions)
         calculate_assessment
       else
         assessments = ReviewResponseMap.get_assessments_for(team)
-        score_team[:scores] = Answer.compute_scores(assessments, questions[:review])
+        score_team[:scores] = Answer.compute_scores(assessments, questions)
```
Comment (Winbobob) [inline app/models/on_the_fly_calc.rb]: I dont think this modification is correct. Please check the `compute_scores` method in `Answer` model.

**Comment by Winbobob** [comment]
With test skeletons.

**Pull request merged**